concepticondata/references/references.bib

```diff
@@ -1,6 +1,15 @@
 % This file was created with JabRef 2.10.
 % Encoding: UTF8
 
+@misc{Bowern2012b,
```

Comment:
If this is the dataset, it was published in 2017, not in 2012. As we already saw that the datasets itself contains HG data plus Swadesh, it is essential to distinguish it from the 2012 publication, at least I am inclined to think about it like this.

Reply:
I agree with you, I went with 2012 because it is the date of the reference. It is almost an ontological question: the conceptlist should be considered a part of the dataset, derived from it, or a new work?

But as said I completely agree with you that 2017 would be better, I myself had some moments of confusion due to the Bowern-2012 already there. WIll happily change it.